framework/assets/yii.js

```diff
@@ -143,74 +144,70 @@ window.yii = (function ($) {
          *             'name2' => 'value2',
          *         ],
          *     ],
-         * ];
+         * ]);
          * ```
          *
          * @param $e the jQuery representation of the element
+         * @param event Related event
          */
         handleAction: function ($e, event) {
             var $form = $e.attr('data-form') ? $('#' + $e.attr('data-form')) : $e.closest('form'),
                 method = !$e.data('method') && $form ? $form.attr('method') : $e.data('method'),
                 action = $e.attr('href'),
+                isValidAction = action && action !== '#',
                 params = $e.data('params'),
-                pjax = $e.data('pjax') || 0,
-                usePjax = pjax !== 0 && $.support.pjax,
-                pjaxPushState = !!$e.data('pjax-push-state'),
```

Comment:
All these variables are actually redundant. No additional manipulations are done with them, so we can do assignment in `pjaxOptions`.